### ISSUE TITLE
Japanese variant of 曜 (and similar kanji) drawn incorrectly

### ISSUE THREAD

**Issue by bionicman52** [parsimonhi/animCJK#21]
From the demo website:
![image](https://user-images.githubusercontent.com/102435129/160255152-705a463c-6631-493c-bcaf-fa36eeb76a36.png)

How it should be:
![image](https://user-images.githubusercontent.com/102435129/160255223-423eecca-d898-44f0-af1c-52a24974187d.png)

Other similar kanji that appear incorrect are 躍, 濯 and 燿. There could be more but those are the ones I found.

**Comment by parsimonhi** [comment]
Hello,

Thanks for your report.

I am not sure to catch what is wrong in animCJK about 曜.

The stroke order seems correct.

And the look seems possible in kaisho style (楷書体) too. See for instance https://www.syotai.net/2012-2-18/

Are you sure that there is an error in animCJK? And what is this error exactly?

**Comment by alpacaStacka** [comment]
Related to the original post so I'll just post here instead of starting a new thread. 

There are a handful of cards that've been caught with mostly, but not entirely minor errors. These range from extra serifs to strokes initiating from the wrong direction (e.g., stroke 13 in the kanji OP was talking about is facing the wrong direction; strokes 6/7 and 9/10 are also highly stylized, though that's less of an issue to me) to outright missing or extra strokes.  If you start at where this link picks up at and scroll down you can see a few of them: https://forums.ankiweb.net/t/jo-makos-shared-deck/422/51?u=omegoa. If you need, I can work with a couple of the other users who have caught problems to make things a bit more presentable.

**Comment by parsimonhi** [comment]
@alpacaStacka

> There are a handful of cards that've been caught with mostly, but not entirely minor errors. These range from extra serifs to strokes initiating from the wrong direction (e.g., stroke 13 in the kanji OP was talking about is facing the wrong direction; strokes 6/7 and 9/10 are also highly stylized, though that's less of an issue to me; there are entries that have strokes initiating from the wrong end, as in stroke 5 of 考) to outright missing or extra strokes.

Since a Chinese font is used as the raw material, there are of course many small differences from the Japanese style. I have already corrected several thousand of these differences. But there are still some left.

> If you start at where this link picks up at and scroll down you can see a few of them: https://forums.ankiweb.net/t/jo-makos-shared-deck/422/51?u=omegoa. If you need, I can work with a couple of the other users who have caught problems to make things a bit more presentable.

Very useful link! I will try to correct all the issues reported there.

Feel free to report any other issues you find.

Note: some errors reported in https://forums.ankiweb.net/t/jo-makos-shared-deck/422/51?u=omegoa are not necessarily errors. For instance, 贈 (svgsJa/64101.svg, jinmeyo kanji) and 贈 (svgsJa/36104.svg, joyo kanji) exist in Japanese and are both in animCJK.

**Comment by parsimonhi** [comment]
I checked errors reported at https://forums.ankiweb.net/t/jo-makos-shared-deck/422

I just updated the master after doing the following:
- change of Japanese glyph of 曜濯燿耀躍拐考条督微称弥祢顧与写判酸新薪親述術垂唾睡錘鬱
- change of Japanese stroke order of 堕謹僅謹勤

More details below about errors reported at https://forums.ankiweb.net/t/jo-makos-shared-deck/422 :

Remember that animCJK is in kaisho style!!! Some reported errors are not errors in kaisho style.
Many other reported errors are in fact a confusion between some characters and their CJK Compatibility versions (CJK compatibility versions have a unicode such as 64xxx, see https://en.wikipedia.org/wiki/CJK_Compatibility_Ideographs for more details on CJK compatibility characters).

拐: 8th stroke is as in 刀, error in animCJK
考: 5th stroke in the wrong direction, error in animCJK
条: bottom component should be as 木, error in animCJK
confusion between 贈(64101) and 贈(36104): error in jo-mako's deck, not in animCJK
confusion between 謁(64098) and 謁(35585): error in jo-mako's deck, not in animCJK
confusion between 賓(64100) and 賓(36051): error in jo-mako's deck, not in animCJK
confusion between 墨(64058) and 墨(22696): error in jo-mako's deck, not in animCJK
confusion between 悔(64061) and  悔(24724): error in jo-mako's deck, not in animCJK
督: 4th stroke seems hooked in Japanese, error in animCJK
隹 (and characters that include 隹): 3rd stroke in kaisho style is drawn from the top-left to the bottom-right, no error in animCJK
曜, 濯, 燿, 耀, 躍: wrong top right component, error in animCJK
覧: 10th stroke horizontal in Japanese? Not sure in kaisho style, pending
confusion between 懲(64064) and 懲(25074): error in jo-mako's deck, not in animCJK
微: 6th, 7th, 8th strokes wrong in animCJK
称, 弥, 祢 : 2nd stroke of the right part in Japanese is flat at its end, not curved, error in animCJK
茶: 7th not hooked? hooked is possible in kaisho style, no error in animCJK
confusion between 穀(64084) and 穀(31296): error in jo-mako's deck, not in animCJK
笑: fixed sometimes ago in animCJK (2022/06/30)
傑: Chinese version of tree for the bottom right component? not always in kaisho style, pending
顧: 户 or 戸 version of door? error in animCJK
not consistent in animCJK: for instance 雇 was modified as many others that contain , but 顧 still contains 户
when alone, 戸 is probably better to avoid confusion with 户
when as a component, one put 戸 in animCJK to be consistent with 戸 when alone
Note: in kaisho style, 戸 component is often drawn as 户. Drawing 戸 as 户 is not an error in kaisho style.
confusion between 祈(64078) and 祈(31048): error in jo-mako's deck, not in animCJK
与: 3th stroke should be longer, error in animCJK
写: 5th stroke should be longer, error in animCJK
判: 5th stroke should be straight, error in animCJK (a curved stroke as in Chinese is probably not an error in kaisho style)
confusion between 煮(64072) and 煮(29038): error in jo-mako's deck, not in animCJK
confusion between 禍(64082) and 禍(31117) and others that have the 示/礻 radical: error in jo-mako's deck, not in animCJK
織 and others that have the tread radical: correct in animCJK. The tread radical is not drawn in kaisho style as in common fonts.
confusion between 突(64085) and 突(31361): error in jo-mako's deck, not in animCJK
confusion between 暑(64067)and 暑(26257): error in jo-mako's deck, not in animCJK
堕: wrong stroke order (4th and 5th strokes), error in animCJK
confusion between 繁(64089)and 繁(32321): error in jo-mako's deck, not in animCJK
幽: correct in animCJK (no, the vertical strokes on the left and the right are not too short in kaisho style)
酉: correct in animCJK (no, the 6th stroke is not too short in kaisho style)
酸: the 11th stroke should be curve, error in animCJK
should be consistent with 俊唆峻竣駿
豊: correct in animCJK (no, the 7th stroke is not too short in kaisho style)
濫 (see also 覧): 13th stroke horizontal in Japanese? not sure in kaisho style, pending
confusion between 朗(63785) and 朗(26391): error in jo-mako's deck, not in animCJK
新, 薪, 親: bottom left component should be 木 instead of 朩. Error in animCJK.
confusion between 卑(64053) and 卑(21329), 碑(64075)/碑(30865): error in jo-mako's deck, not in animCJK
述: 2nd, 3rd and 4th strokes wrong in animCJK (short oblique are required, straight vertical desired)
術: 6th and 7th strokes wrong in animCJK
should be consistent with 述 (see above)
積: stroke order fixed sometimes ago in animCJK (2022/11/04)
confusion between 謹(64099) and 謹(35641): wrong glyph, error in jo-mako's deck, not in animCJK
謹(64099), 謹(35641): wrong stroke order of the last 4 strokes, error in animCJK
僅(64052): same issue
勤(20677): same issue
confusion between 漢(64071) and 漢(28450): error in jo-mako's deck, not in animCJK
confusion between 嘆(64055) and 嘆(22022): error in jo-mako's deck, not in animCJK
confusion between 難(64104) and 難(38627): error in jo-mako's deck, not in animCJK
垂, 唾, 睡, 錘: last stroke too short? error in animCJK
有 and 右: stroke order correct in animCJK. It is an exception. The first stroke is the oblique one, not the horizontal one.

Hope it helps!
